tools/ports/freetype.py

```diff
@@ -5,56 +5,47 @@
 
 import os
 
-TAG = 'version_1'
-HASH = '0d0b1280ba0501ad0a23cf1daa1f86821c722218b59432734d3087a89acd22aabd5c3e5e1269700dcd41e87073046e906060f167c032eb91a3ac8c5808a02783'
+TAG = '2-12-1'
```

Comment:
Maybe just include the `VER-` prefix here since it seems to be part of the upstream tag name?

Reply:
Done.